package/src/components/MessageInput/MessageInput.tsx

```diff
@@ -500,45 +453,22 @@ const MessageInputWithContext = <
                   />
                 ) : null}
                 {fileUploads.length ? <FileUploadPreview /> : null}
-                <View
-                  style={[
-                    styles.autoCompleteInputContainer,
-                    {
-                      paddingLeft: giphyActive ? 8 : 16,
-                      paddingRight: giphyActive ? 10 : 16,
-                    },
-                    autoCompleteInputContainer,
-                  ]}
-                >
-                  {giphyActive && (
-                    <View
-                      style={[
-                        styles.giphyContainer,
-                        { backgroundColor: accent_blue },
-                        giphyContainer,
-                      ]}
-                    >
-                      <Lightning height={16} pathFill={white} width={16} />
-                      <Text style={[styles.giphyText, { color: white }, giphyText]}>GIPHY</Text>
-                    </View>
-                  )}
-                  <AutoCompleteInput<StreamChatGenerics>
+                {giphyActive ? (
+                  <GiphyCommandInput
                     additionalTextInputProps={additionalTextInputProps}
-                    cooldownActive={!!cooldownRemainingSeconds}
+                    cooldownEndsAt={cooldownEndsAt}
+                    disabled={disabled}
+                    setGiphyActive={setGiphyActive}
+                    setShowMoreOptions={setShowMoreOptions}
```

Comment:
Same here. Check which values can you access from context, and remove them from prop

Reply:
Done